static/app/utils/replays/replay.tsx

```diff
@@ -2,6 +2,7 @@ type JsonObject = Record<string, unknown>;
 type JsonArray = unknown[];
 
 export type NetworkMetaWarning =
```

Comment:
Is this exported from SDK at all?

Reply:
not right now! We can think about exporting this in the future. but probably this doesn't really make sense here, because we want to keep this broader - we want to keep all values that this could ever have here I think? as we also need to make sure to keep covering older versions?